bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDClientImpl.java

```diff
@@ -88,15 +93,38 @@ public BFDClientImpl(IGenericClient bfdFhirRestClient) {
             exclude = { ResourceNotFoundException.class }
     )
     public Bundle requestEOBFromServer(String patientID) {
+        return requestEOBFromServer(patientID, null);
+    }
+
+    /**
+     * Queries Blue Button server for Explanations of Benefit associated with a given patient
+     * similar to {@link #requestEOBFromServer(String)} but includes a date filter in which the
+     * _lastUpdated date must be after
+     * <p>
+     *
+     * @param patientID The requested patient's ID
+     * @param sinceTime The start date for the request
+     * @return {@link Bundle} Containing a number (possibly 0) of {@link ExplanationOfBenefit}
+     * objects
+     * @throws ResourceNotFoundException when the requested patient does not exist
+     */
+    @Override
+    @Retryable(
+            maxAttemptsExpression = "${bfd.retry.maxAttempts:3}",
+            backoff = @Backoff(delayExpression = "${bfd.retry.backoffDelay:250}", multiplier = 2),
+            exclude = { ResourceNotFoundException.class }
+    )
+    public Bundle requestEOBFromServer(String patientID, OffsetDateTime sinceTime) {
         var excludeSAMHSA = new TokenClientParam("excludeSAMHSA").exactly().code("true");
-        return client.search()
+        IQuery<IBaseBundle> query = client.search()
                 .forResource(ExplanationOfBenefit.class)
                 .where(ExplanationOfBenefit.PATIENT.hasId(patientID))
-                .and(excludeSAMHSA)
-                .count(pageSize)
-                .returnBundle(Bundle.class)
-                .encodedJson()
-                .execute();
+                .and(excludeSAMHSA);
+        if (sinceTime != null) {
+            Date d = Date.from(sinceTime.toInstant());
+            query.lastUpdated(new DateRangeParam(d, null));
+        }
```

Comment:
This `if` block breaks the method chaining that was flowing so nicely :) You can move it to the very beginning of the method, set a `lastUpdated` variable or something, and then just add `.lastUpdated(lastUpdated` to the chain (it accepts nulls).